src/python_routing_v02/fast_reach/mc_reach.pyx

```diff
@@ -861,30 +864,48 @@ cpdef object compute_network_structured_obj(
               buf_view[i, 10] = flowveldepth[segment.id, timestep-1, 0]
               buf_view[i, 11] = 0.0 #flowveldepth[segment.id, timestep-1, 1]
               buf_view[i, 12] = flowveldepth[segment.id, timestep-1, 2]
-
+            """
+            for i, segment in enumerate(r):
```

Comment:
I  like the simplicity of implementing reach as a container type (list) for segments with __get_item__ but it probably needs some additional documentation.